parl/remote/master.py

```diff
@@ -57,9 +57,10 @@ class Master(object):
         port: The ip port that the master node binds to.
     """
 
-    def __init__(self, port):
+    def __init__(self, port, monitor_url):
```

Comment:
Why don't we just use `monitor_port` instead of `monitor_url`?

Reply:
Change to use `monitor_port`